Выполняю задание 8

LexyJexon opened the pull request:
Логику написал , неуспел отладить

LexyJexon committed:
Выполняю задание 8

Логику реалищовал , неуспел отладить

LexyJexon commented:
не могу понять в чём ошибка 
![image](https://user-images.githubusercontent.com/76475130/197282434-48a085e7-a1bd-4830-9a48-9a81600b7f5f.png)

LexyJexon committed:
Дорабатываю задание, избавился от структуры комбинаций карт, доработал поиск комбинаций и их филтрацию от дублей

Использование ссылки на структуру как тип для карточной комбинации вызывало ошибку адресов, с которой не разобрался
Поиск комбинаций выдавал только комбинации из 5 одинаковых карт для каждой из карт в файле, необходимо было проверять на повторение карт в комбинациях

LexyJexon committed:
Добавил библиотеку для поиска комбинаций карт

Собственно написанный алгоритм генерации всех возможных комбинаций карт возвращал слишком мало комбинаций, то есть некорректно отрабатывал

lesmiras reviewed (commented):
1) Можно попровать разграничить генерацию исходных данных и результата отдельными командами в корневом main.go файла, вместо то того чтобы создавать новый main.go в другой папке. Желательно иметь только 1 main.go файл.
2) В результатах много дублирующих комбинации.
Не получилось убрать дубликаты?

LexyJexon committed:
Исправил поиск дублей карт в файле и добавил считвание флагов вызова команды в коммандной строке

Некорректно отрабатывал поиск дубликатов карт в исходном наборе, из-за чего получалось куча дублей комбинаций. Переименновал второй main.go файл в parse.go, в пакете parse_combinations, и вызываю его в main.go с помощью считывания фдака -command при вызове go run
